package.json

```diff
@@ -45,7 +45,7 @@
     "@babel/preset-env": "^7.10.2",
     "@babel/preset-react": "^7.10.1",
     "@octokit/graphql": "^4.5.0",
-    "@octokit/rest": "^17.11.0",
+    "@octokit/rest": "^18.0.3",
```

Comment:
Do we want to pin to a minor version (~18.0.3), in case of future breaks?

Reply:
I'd rather leave as-is so we handle the versions through the CLI instead of handcoding just one. This breaking change was because they moved to a more ESM-friendly format, so I don't think we'll go through it again.